target-app/Networking/Services/TargetServices.swift

```diff
@@ -43,4 +43,23 @@ class TargetServices {
         }
     }
     
+    class func get(
```

Comment:
```suggestion
    class func getTarget(
```

Reply:
@atorrico09 this is what I meant with the endpoint name being confusing. This is what I thought as well the first time, but later realised that it is to fetch all the targets.

Reply:
renamed it to `getAll`, as @glm4 suggested below